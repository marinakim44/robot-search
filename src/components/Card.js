import React, { Component } from "react";

export default class Card extends Component {
  render() {
    return (
      <div className="tc bg-light-green grow br3 pa3 ma2 dib bw2 shadow-5">
        <img src={`https://robohash.org/${this.props.id}`} />
        <h2>{this.props.name}</h2>
        <p>{this.props.email}</p>
      </div>
    );
  }
}
