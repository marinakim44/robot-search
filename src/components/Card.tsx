import { Monster } from "../containers/App";

export default function Card({ id, name, email }: Monster) {
  return (
    <div className="tc bg-light-green grow br3 pa3 ma2 dib bw2 shadow-5">
      <img src={`https://robohash.org/${id}?size=200x200`} alt={name} />
      <h2>{name}</h2>
      <p>{email}</p>
    </div>
  );
}
