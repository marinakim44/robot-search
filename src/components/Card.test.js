import { shallow } from "enzyme";
import Card from "./Card";
import React from "react";

it("expect to render card component", () => {
  expect(shallow(<Card />)).toMatchSnapshot();
});
