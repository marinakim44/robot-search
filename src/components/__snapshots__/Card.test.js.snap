// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render card component 1`] = `
<div
  className="tc bg-light-green grow br3 pa3 ma2 dib bw2 shadow-5"
>
  <img
    src="https://robohash.org/undefined?size=200x200"
  />
  <h2 />
  <p />
</div>
`;
